Python/pylifecycle.c

```diff
@@ -1477,20 +1478,40 @@ Py_SetPythonHome(wchar_t *home)
 }
 
 wchar_t *
-Py_GetPythonHome(void)
+_Py_GetPythonHomeWithConfig(const _PyMainInterpreterConfig *config)
```

Comment:
I'm not sure I actually like this API design, but since it's an internal API, I don't have any obviously better ideas, and we expect further refactoring of this code in the future, I'm also not going to quibble over it.